<launch>
  <!-- Define arguments: -->
  <!--
  <arg name="use_gripper" default="true"  doc="Should the gripper be used?" />  # example
  -->

  <!-- Define parameters: -->
  <!--
  <param name="param_name"  value="param_value" />  # example
  <param name="m_ee" type="double" value="0.76" if="$(arg use_gripper)" />  # example
  -->

  <!-- Load configuations into parameter server -->
  <rosparam file="$(find irobman_project)/config/general_config.yaml" command="load" />
  
  <!-- Launch nodes -->
  <node name="pose_estimation" pkg="irobman_project" type="pose_estimation_node" output="screen" />
  <node name="motion_planner" pkg="irobman_project" type="motion_planner.py" output="screen" />
  <node name="cube_detection" pkg="irobman_project" type="cube_detection.py" output="screen" />
  <node name="capture_image" pkg="irobman_project" type="capture_image.py" output="screen" />
  <!--<node name="capture_image" pkg="pose_estimation" type="capture_image_node" output="screen" />-->
</launch>
